tests/test_ibs_equilibrium_emittances.py

```diff
@@ -25,7 +25,7 @@ def test_ibs_emittance_constraints(emittance_constraint):
 
     emittance_coupling_factor = 0.02
 
-    time, emittances_x_list, emittances_y_list, emittances_z_list, T_x, T_y, T_z = xf.ibs.compute_emittance_evolution(
+    time, emittances_x_list, emittances_y_list, emittances_z_list, T_x, T_y, T_z = xf.ibs.compute_equilibrium_emittances_from_sr_and_ibs(
```

Comment:
Automatic update of this and below but I will have to update the tests anyway as now we return a Table.